synapse/lib/cmdr.py

```diff
@@ -13,28 +20,65 @@
         s_cmds_cron.At,
         s_cmds_cron.Cron,
         s_cmds_cortex.Log,
+        s_cmds_boss.PsCmd,
+        s_cmds_boss.KillCmd,
         s_cmds_hive.HiveCmd,
-        s_cmds_cortex.PsCmd,
-        s_cmds_cortex.KillCmd,
         s_cmds_cortex.StormCmd,
         s_cmds_trigger.Trigger,
     ),
 }
 
-async def getItemCmdr(cell, outp=None, **opts):
+
+cmdsbyclass = {
```

Comment:
This has historically bitten us in the past from a backward-compatibility perspective.  Any instance of wanting to rename or reorganize "internal" code layout results in a backward-incompatible change.  We need the abstraction that is not specifically tied to code layout.

Reply:
We could use strip off and use the class names

Reply:
this has been changed to use the class name instead of the full path